src/lib.rs

```diff
@@ -69,23 +126,21 @@ impl UnixListener {
     ///
     /// The returned listener will be associated with the given event loop
     /// specified by `handle` and is ready to perform I/O.
-    pub fn from_listener(listener: net::UnixListener, handle: &Handle)
-                         -> io::Result<UnixListener> {
+    pub fn from_listener(listener: net::UnixListener, handle: Handle) -> io::Result<UnixListener> {
```

Comment:
This does not seem related to the PR? Also, this probably should be `&Handle`.

Also, could this be named `from_std`?

Reply:
> This does not seem related to the PR? Also, this probably should be &Handle

A `Handle` is needed to construct the `PollEvented` that wraps the mio stream so I had to take ownership and store a handle in the listener for that. Figured it were better to be explicit about the owning part.